.github/workflows/check-pr-title.yaml

```
@@ -0,0 +1,26 @@
+name: "Check PR title"
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+
+jobs:
+  check-title:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if the PR title is well dressed
+        env:
+          PR_TITLE: ${{ github.event.pull_request.title }}
+          REGEX: '^(((build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test){1}(\([\w\-\.]+\))?(!)?)|((ROX-[0-9]+,)*ROX-[0-9]+))(: .*)$'
```

AFAIR @porridge requested to split it into multiline 
```suggestion
          REGEX: >
            ^(((build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test){1}
           (\([\w\-\.]+\))?(!)?)|
           ((ROX-[0-9]+,)*ROX-[0-9]+))(: .*)$
```

---

I'd even go as far as:

```suggestion
          REGEX: >
            ^(
               (
                (build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test){1}
                (\([\w\-\.]+\))?(!)?
               )
               |
               (
                (ROX-[0-9]+,)*ROX-[0-9]+
               )
              )(: .*)$
```

---

This will put spaces in place of `\n`, which will corrupt the regex.